docs/users.rst

```diff
@@ -26,20 +41,157 @@ image owners can use ``koji watch-logs <task id> | grep -w x86_64``.
 Configuring koji-containerbuild for koji CLI
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+To be able to build container images from CLI `koji-containerbuild plugin`_
+must be installed.
+
+To configure CLI you'll need to copy [koji] section in /etc/koji.conf to
+[koji-containerbuild] and optionally adapt configuration there.
+
+.. _`koji-containerbuild plugin`: https://github.com/containerbuildsystem/koji-containerbuild
+
+
+Koji Build Results
+~~~~~~~~~~~~~~~~~~
+
+Koji Web
+********
+
+This is the easiest way to access information about OSBS builds.
+
+List Builds
++++++++++++
+
+Navigate to the “Builds” tab in brew and set the “Type” filter to ``image``.
+
+
+Get Build
++++++++++
+
+If you have the build ID, got to https://<KOJI_WEB_URL>/buildinfo?buildID=<build-ID>
+
+If you want to search by prefix, use the search box on top of the page.
+For example, redis-* and select “Builds”.
+
+This will also show the tarball docker image produced by build.
+
+Click on “info” to view the archive information. In the “Extra” section,
+docker.repositories specifies the pull specification for the build.
+This includes both the tags applied to the image and the manifest digest for
+the version-release tag. Additionally, the image ID for the image is available
+in docker.id, and this is a Docker-1.10-style content-addressable image ID
+for newly-built images.
+
+
+Also in the "Extra" section, docker.config shows (parts of)
+the `docker image JSON description`_, as well it indicates container image
+API version.
+
+
+See atomic-reactor documentation for
+a `full description of the Koji container image metadata`_.
```

Comment:
maybe delete this line? since you have the same on line 94?

Reply:
line 94 is URL target for line 91, I cannot remove it, it won't work as hyperlink

Reply:
ahh ok